pkg/settings/migration.go

```diff
@@ -1,19 +0,0 @@
-package settings
```

Comment:
+1 on removing this part of settings, no longer needed since rollback will always be on-demand when user requests it.